src/templates/ratings/add.html

```diff
@@ -7,6 +7,12 @@
           {% endif %}>
       {% if isModal %}
         <h3>{{ existingReview and _('Edit Review') or _('Leave a Review') }}</h3>
+	<p class="text-small">
+           {{ _('Please read the <a href="https://developer.mozilla.org/en-US/Marketplace/User_Review_Guidelines">Review Guidelines</a> for more details about rating apps.
```

Comment:
I noticed this URL has `/en-US/` in it, which is for only US English speakers; I would recommend changing `https://developer.mozilla.org/en-US/Marketplace/…` to `"https://developer.mozilla.org/Marketplace/…`.

also, it's good practice to put the URL outside of the localised string - so if the URL changes, localisers don't have to localise this string in every language again.

check out this example: https://github.com/mozilla/fireplace/blob/d0ceaa0e4/src/templates/settings.html#L31